sdk/src/main/kotlin/com/processout/sdk/api/repository/BaseRepository.kt

```diff
@@ -26,23 +27,23 @@ internal abstract class BaseRepository(
                     ?: ProcessOutResult.Failure(
                         POFailure.Code.Internal(),
                         "Response body is empty."
-                    )
+                    ).also { POLogger.error("%s", it) }
```

Comment:
I suggest to add proper message alongside every log call. When we will analyse those messages, plain error description without any context when and why it did happen will be more complex to understand.

For example `POST: api/card (requestId="123456") method did fail with error: Wrong card number...` is easier to analyse than `Wrong card number`

Reply:
Updated